### ISSUE TITLE
Velero tries to snapshot PVCs not connected to pods

### ISSUE THREAD

**Issue by sseago** [fusor/mig-controller#281]
I'm not sure what the actual bug/resolution looks like here, but it looks like if there are PVCs in the migration namespace that aren't connected to running pods, velero will attempt to copy them by snapshot. The reason for this is that restic will operate on the PVC if there's a pod with a restic annotation that references it, so from velero's point of view, an unconnected (but bound) PVC looks the same as a pod where we've explicitly decided to use snapshots to copy -- either way, there's no restic annotation so it attempts to back up (and restore) by snapshot.

**Comment by sseago** [comment]
Actually, this may be a false alarm. In digging into the Controller annotation code, unconnected PVCs shouldn't even get the IncludedInStageBackupLabel applied. In the case that I observed the attempted snapshot, the stage pod wasn't there to tell Restic to copy the volume, but that was due to a src cluster problem. The PVCs *were* connected to the (non-stage) pod, so they got the annotation. I suspect a truly unconnected PVC won't be snapshotted at all since it won't be in the backup at all. Leaving the issue open until I confirm it by migrating a namespace with an unconnected (but bound) PVC.
